BepInEx.Core/Contract/PluginInfo.cs

```diff
@@ -33,24 +38,23 @@ public class PluginInfo : ICacheable
     /// </summary>
     public IEnumerable<BepInIncompatibility> Incompatibilities { get; internal set; }
 
-    /// <summary>
-    ///     File path to the plugin DLL
-    /// </summary>
-    public string Location { get; internal set; }
```

Comment:
There may be some trouble caused by this, some plugins relied on this to find their bundled resources. Not sure how to resolve this though, especially when loading from something like a zip file.

Reply:
As discussed, added a GetFile method that plugins can call